src/commands.ts

```diff
@@ -36,15 +37,21 @@ export function loadCommands(shell: Shell): Map<string, Command> {
             usage: "cl[ear]",
             execute: async () => shell.t.clear()
         },
+        close: {
+            name: "close",
+            help: "Close the current gate",
+            usage: "cl[ose]",
```

Comment:
We have both "close" and "clear" beginning with "cl" ao it should be "clo[se]" and the same for "cle[ar]"

Reply:
I'll try to make that automatic happen when printing `help`.